configuration/concepts/laborders.csv

```diff
@@ -0,0 +1,85 @@
+	Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:km,Short name:en,Short name:km,Description:en,Description:km,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1150
+,,,abn,abn,,,,,Abnormal,Boolean,,,,,,,,,,,
+,,,HBsAg Confirm (Serum),,,,,,LabTest,Text,,,,,,,,,,,
+,,,HBs Antibody (Serum),,,,,,LabTest,Text,,,,,,,,,,,
+,,,HIV Determine (Serum),,,,,,LabTest,Text,,,,,,,,,,,
+,,,HIV Serodia (Serum),,,,,,LabTest,Text,,,,,,,,,,,
+,,,TP Syphilis (Serum),,,,,,LabTest,Text,,,,,,,,,,,
+,,,Widal Test To,,,,,,LabTest,Text,,,,,,,,,,,
+,,,Widal Test Th,,,,,,LabTest,Text,,,,,,,,,,,
+,,,Rheumatoid Factor,,,,,,LabTest,Numeric,,,,TRUE,iu/ml,,20,,,,
+,,,Rheumatoid Factor Data,,Rheumatoid Factor[<20],,,,Concept Details,N/A,,,Rheumatoid Factor; abn;,,,,,,,,
+,,,ASO Titer,,,,,,LabTest,Numeric,,,,TRUE,iu/l,,200,,,,
+,,,ASO Titer Data,,ASO Titer[<200],,,,Concept Details,N/A,,,ASO Titer; abn;,,,,,,,,
+,,,SLE Antibody Latex Test,,,,,,LabTest,Text,,,,,,,,,,,
+,,,Demo of Latex Test,,,,,,LabTest,Text,,,,,,,,,,,
+,,,Dimertest,,,,,,LabTest,Numeric,,,,TRUE,mg/l,,0.2,,,,
+,,,Dimertest Data,,Dimertest(plasma)[<0.2],,,,Concept Details,N/A,,,Dimertest; abn;,,,,,,,,
+,,,Dengue IgG,,,,,,LabTest,Coded,,Cambodia_Positive; Cambodia_Negative,,,,,,,,,
+,,,Dengue IgM,,,,,,LabTest,Coded,,Cambodia_Positive; Cambodia_Negative,,,,,,,,,
+,,,Pastorex Cryptococcus,,,,,,LabTest,Text,,,,,,,,,,,
+,,,TB Hexagon,,[Senior Dr only] TB Hexagon,,,,LabTest,Text,,,,,,,,,,,
+,,,Sodium,,,,,,ChemTest,Numeric,,,,TRUE,mmol/l,136,148,,,,

```

Comment:
Here you seem to use a Concept Class "ChemTest". Where does this concept class come from?